internal/controller/pkg/revision/deployment_test.go

```diff
@@ -0,0 +1,159 @@
+package revision
+
+import (
+	"testing"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	pkgmetav1 "github.com/crossplane/crossplane/apis/pkg/meta/v1"
+	v1 "github.com/crossplane/crossplane/apis/pkg/v1"
+	"github.com/crossplane/crossplane/apis/pkg/v1alpha1"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
```

Comment:
The linter comment is a little vague here; it's asking you to separate out all the `crossplane/crossplane` (i.e. "local") imports into a group separated by a line of whitespace. We typically separate out runtime imports too, so it looks something like:

```go
import (
	"stdlib-imports"

	"third-party-imports"

	"crossplane-runtime-imports"

	"crossplane-imports"
)

Reply:
:+1: 

forgot `make reviewable` after adding the test.

Running `goimports -w -local github.com/crossplane/crossplane internal/controller/pkg/revision/deployment_test.go` now